src/Tickets/Emails/Email_Template.php

```diff
@@ -0,0 +1,193 @@
+<?php
+/**
+ * Handles registering and setup for the Tickets Emails settings tab.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+
+namespace TEC\Tickets\Emails;
+
+use Tribe__Template;
+use Tribe__Tickets__Main;
+
+/**
+ * Class Email_Template
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+class Email_Template {
+
+	private bool $preview = false;
+
+	private Array $context_data = [];
+	
+	/**
+	 * Gets the template instance used to setup the rendering html.
+	 *
+	 * @since TBD
+	 *
+	 * @return Tribe__Template The template object.
+	 */
+	public function get_template(): Tribe__Template {
+		if ( empty( $this->template ) ) {
+			$this->template = new Tribe__Template();
+			$this->template->set_template_origin( Tribe__Tickets__Main::instance() );
+			$this->template->set_template_folder( 'src/admin-views/settings/emails' );
+			$this->template->set_template_context_extract( true );
+		}
+
+		return $this->template;
+	}
+
+	/**
+	 * Returns the email template HTML.
+	 *
+	 * @return string The HTML of the template.
+	 */
+	public function get_html() {
```

Comment:
We can possibly add the return type.

Reply:
There is a `@return string...` statement already. Am I misunderstanding?

Reply:
Normally the linter is also highlighting if there's no var name. But it's also good.